src/System.Memory/ref/System.Memory.cs

```diff
@@ -199,7 +199,6 @@ public abstract partial class ReadOnlySequenceSegment<T>
     public readonly partial struct ReadOnlySequence<T>
     {
         private readonly object _dummy;
-        private readonly int _dummyPrimitive;
```

Comment:
I am surprised we are dropping this here, especially since we are not dropping it in `SequencePosition`.

@ericstj, do you know what changed? Is this expected?

Reply:
It might be me trying to merge in incorrectly. I'll take another look.

Reply:
I ran it again and it didn't change it.

Reply:
The field should be there according to the rules in https://github.com/dotnet/arcade/blob/5e7ce5b394f3477bb0a485a4b761b7742e95be37/src/Microsoft.Cci.Extensions/Writers/CSharp/CSharpWriter.cs#L153 . It looks suspect to me that this change is dropping it.

Reply:
Manually put it back

Reply:
Manually put it back

Reply:
How are you generating the reference sources?

Reply:
/cc @safern

Reply:
Via the command line target. `dotnet msbuild /t:GenerateReferenceSource` in the ref folder.

Reply:
I think you need to run from the source folder, CC. @ericstj

Reply:
right, plus this will benefit from @safern's feature that will follow type-forwards in implementation.

Reply:
> Via the command line target. `dotnet msbuild /t:GenerateReferenceSource` in the ref folder.

> I think you need to run from the source folder

I think @JeremyKuhne may have misspoken (please correct me if I am mistaken). Running from the ref folder would **not** have made such changes (and it doesn't). I believe he **did** run from the source folder and it is dropping the primitive type dummy field. I just tried it myself locally on corefx master.

Running the following:
> E:\GitHub\Fork\corefx\src\System.Memory\src>dotnet msbuild /t:GenerateReferenceSource
Microsoft (R) Build Engine version 16.1.27-preview+g91af6445a6 for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.

>   Unable to resolve assembly 'Assembly(Name=System.Private.CoreLib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)' referenced by 'Assembly(Name=System.Memory, Version=4.2.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51)'.

Results in:
![image](https://user-images.githubusercontent.com/6527137/57960347-1a7c0c00-78bd-11e9-9e38-7bb62acc8194.png)